ansible/roles/vm_set/templates/mux-simulator.service.j2

```diff
@@ -3,4 +3,4 @@ Description=mux simulator
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/env python {{ abs_root_path }}/mux_simulator.py {{ mux_simulator_port }}
+ExecStart=/usr/bin/env python {{ abs_root_path }}/mux_simulator.py {{ mux_simulator_port }} {{ vm_set_name }} -v
```

Comment:
Do we need to make ```-v``` as default?

Reply:
I think with `-v` as default, it would be easier to debug mux simulator issue observed during nightly tests. The `-v` option won't flood the mux simulator log with too much content to make it unreadable.
Based on these considerations, I decided to make `-v` as default.